_posts/2020-03-24-dask-developer-workshop.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: single
+title:  "Karlsruhe to D.C. ― a Dask story"
+date:   2020-03-24 20:00:00 +0100
+tags: technology python data-engineering Dask distributed
+header:
+  overlay_image: assets/images/dask_icon.svg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Lucas Rademaker & Nefta Kanilmaz
+author_profile: true
+---
+# Karlsruhe to D.C. ― a Dask story
+
+Back in February 2020, we (Florian Jetter, Nefta Kanilmaz and Lucas Rademaker) travelled to the Washington D.C. metropolitan area to attend the first Dask developer conference. Our primary goal was to discuss Blue Yonder’s issues related to Dask with the attending developers and users. When we returned back to Germany, we had not only connected with many of the core developers in the Dask community, but also had an (almost finished) implementation of a distributed semaphore in our bags. 
+
+There is [a blog post](https://blog.dask.org/2020/04/28/dask-summit) giving an overview on and summary of the workshop talks. We want to share our experience around this workshop. 
+
+## Setting
+
+The three-day long workshop included sessions with short talks, as well as time slots for working on Dask-related issues and discussions.
+Talks covered a broad spectrum of topics including Dask-ML and usage of Dask in general data analysis, Dask deployment and infrastructure, and many more. All talks focused on answering these key questions: 
+
+- Who are Dask users and what are their use cases?
+- What are current pain points and what needs to be fixed as soon as possible?
+- What is on the Dask user’s wish list?
+
+In other words, this was the chance for presenters to officially “rant” [*] about Dask, with experts having the experience and knowledge to help in the same room.
+This exchange between users and developers enabled immediate fixes of minor problems, identifying synergies between different projects, as well as shaping the roadmap for future development of Dask. We feel that this is a successful model for driving the Dask open-source community.
+
+[*] quote Matthew Rocklin
```

Comment:
The tenses in the quote appear to be messy. Is it a verbatim quote?

Reply:
@nefta-kanilmaz-jdas

Reply:
rendering issue: 
![image](https://user-images.githubusercontent.com/57136044/83024601-ecccb580-a02e-11ea-8745-1387e3f65bd3.png)